extra/alerts/pcmk_alert_sample.sh

```diff
@@ -56,12 +61,12 @@ case $CRM_notify_kind in
 	case ${CRM_notify_desc} in
 	    Cancelled) ;;
 	    *)
-		echo "Resource operation '${CRM_notify_task}${CRM_notify_interval}' for '${CRM_notify_rsc}' on '${CRM_notify_node}': ${CRM_notify_desc}${CRM_notify_target_rc}" >> ${CRM_notify_recipient}
+		echo "${tstamp}Resource operation '${CRM_notify_task}${CRM_notify_interval}' for '${CRM_notify_rsc}' on '${CRM_notify_node}': ${CRM_notify_desc}${CRM_notify_target_rc}" >> ${CRM_notify_recipient}
 		;;
 	esac
 	;;
     *)
-        echo "Unhandled $CRM_notify_kind notification" >> ${CRM_notify_recipient}
+        echo "${tstamp}Unhandled $CRM_notify_kind notification" >> ${CRM_notify_recipient}
 	env | grep CRM_notify >> ${CRM_notify_recipient}
         ;;
 
```

Comment:
Great, let's also move pcmk_snmp_helper.sh to the alerts directory (and update the comments to reflect the new configuration).